apps/aecore/lib/aecore/chain/chainstate.ex

```diff
@@ -71,8 +71,10 @@ defmodule Aecore.Chain.Chainstate do
     chainstate_with_coinbase =
       calculate_chain_state_coinbase(txs, chainstate, block_height, miner)
 
+    chainstate_with_pruned_calls = Call.reset_calls(chainstate_with_coinbase, block_height)
```

Comment:
if the variable name is `_pruned_calls` the function should be `prune_calls` maybe